README.md

```diff
@@ -212,6 +212,7 @@ Questions can be asked by raising an `Issue`.
 - [Phurba Gyalzen Sherpa](https://github.com/PhurbaGyalzen/)
 - [Pradhumn Dave](https://github.com/davedevelopers)
 - [Prathamesh Giri](https://github.com/Pratham31)
+- [Prathamesh Borse](https://github.com/viraldevpb)
```

Comment:
Shouldn't this entry be above Prathamesh Giri to make sure it's alphabetical order?